src/main/resources/com/toptrumps/online/scripts/plugins/message.js

```diff
@@ -0,0 +1,33 @@
+const Message = (($) => {
+    const gameStatusSelector = '.js-game-status-content';
+    const gameStatusLogSelector = ".js-game-log";
+    const gameLetterSelector = ".game-status__letter";
+    const gameLetterClass = "game-status__letter";
+
+    const animateLog = () => {
```

Comment:
I think the use of the name "log" here can cause some confusion (if you take into account the CLI's logger). I'd suggest ```infoPanel```.